### PR TITLE
Rename Basic to Local

### PR DESCRIPTION
**Pull request by raraveen**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs [hunk 1]
```diff
@@ -61,6 +61,7 @@ public class NewAzureExpressRouteCircuitCommand : ExpressRouteCircuitBaseCmdlet
             MNM.ExpressRouteCircuitSkuTier.Standard,
             MNM.ExpressRouteCircuitSkuTier.Premium,
             MNM.ExpressRouteCircuitSkuTier.Basic,
```
Comment (number213) [inline src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs]: Do you need to communicate to customers that `Basic` is being deprecated?

Comment (raraveen) [reply src/Network/Network/ExpressRouteCircuit/NewAzureExpressRouteCircuitCommand.cs]: Yes, the PMs will communicate this broadly. We have 2 customers using Basic, hence keeping it here. We will work with the 2 customers to move to Local.

File: src/Network/Network/help/New-AzExpressRouteCircuit.md [hunk 2]
```diff
@@ -298,13 +298,13 @@ Accept wildcard characters: False
 ```
 
 ### -SkuTier
-The tier of service for the circuit. Possible values for this parameter are: `Standard`, `Premium` or `Basic`.
+The tier of service for the circuit. Possible values for this parameter are: `Standard`, `Premium` or `Local`.
```
Comment (MikhailTryakhov) [inline src/Network/Network/help/New-AzExpressRouteCircuit.md]: so you're replacing this value here but in NewAzureExpressRouteCircuit command you're adding this value, not replacing. can you please confirm?

File: src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs [hunk 3]
```diff
@@ -97,5 +97,13 @@ public void TestExpressRouteCircuitPeeringWithRouteFilter()
         {
             TestRunner.RunTestScript("Test-ExpressRouteCircuitPeeringWithRouteFilter");
         }
-    }
+
+	    [Fact(Skip = "Rerecord tests")]
```
Comment (MikhailTryakhov) [inline src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs]: Can you re-record test without skipping? And/or sync up with @srividyathirumala what to do with that?

File: src/Network/Network/help/New-AzExpressRouteCircuit.md (hunk 2 above)
Comment (raraveen) [reply src/Network/Network/help/New-AzExpressRouteCircuit.md]: That is correct.

File: src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs (hunk 3 above)
Comment (raraveen) [reply src/Network/Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs]: Removed the skip. @number213 had recorded yesterday.

**Comment by number213** [comment]
For some reason CI doesn't start. @cormacpayne could you please take a look?

**Pull request merged**